tests/test_state_suite.py

```
@@ -1,28 +1,57 @@
+
+
+def _run(coro):
+    """Run an async event handler synchronously."""
+    return asyncio.run(coro)
+
+
+_PARENT_FIELDS = dict(
+    suites=[], suite=None, children=[], child=None,
+    cases_for_search=[], show_case_search=False, search_case_value="",
+    scenarios_for_search=[], show_scenario_search=False, search_scenario_value="",
+)
```

Ruff (0.15.7)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)